spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/data/aerospike/AerospikeReactiveDataConfiguration.java

```diff
@@ -69,8 +71,16 @@ public ReactorIndexRefresher reactiveAerospikeIndexRefresher(AerospikeReactorCli
                                                                  IndexesCacheUpdater indexesCacheUpdater) {
         ReactorIndexRefresher refresher = new ReactorIndexRefresher(aerospikeReactorClient, aerospikeReactorClient.getInfoPolicyDefault(),
                 new InternalIndexOperations(new IndexInfoParser()), indexesCacheUpdater);
-        // NOTE: Do not call refresh indexes on startup for ReactorIndexRefresher, since IndexRefresher will do that
+        refresher.refreshIndexes();
```

Comment:
so comment said you should not call it ;D

Reply:
because contexts were shared